### PR TITLE
net-irc/polari: Bump to 46.0

### PR DESCRIPTION
**Pull request by flint2**
net-irc/polari: Bump to 46.0

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: net-irc/polari/polari-46.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="An IRC client for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Polari https://gitlab.gnome.org/GNOME/polari"
```
Comment (Jamim) [inline net-irc/polari/polari-46.0.ebuild:9]: https://wiki.gnome.org/Apps/Polari is planned for retirement, please use https://apps.gnome.org/Polari instead.
https://gitlab.gnome.org/GNOME/polari/-/blob/46.0/README.md?plain=1#L38
```suggestion
HOMEPAGE="https://apps.gnome.org/Polari https://gitlab.gnome.org/GNOME/polari"
```

File: net-irc/polari/polari-46.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	net-libs/telepathy-glib[introspection]
+	>=dev-libs/gobject-introspection-1.50:=
+	>=dev-libs/gjs-1.73.1
+
+	x11-libs/gdk-pixbuf:2[introspection]
+	>=app-text/gspell-1.4.0[introspection]
+	x11-libs/pango[introspection]
+	app-crypt/libsecret[introspection]
+	net-libs/libsoup:3.0[introspection]
+	net-im/telepathy-logger[introspection]
```
Comment (Jamim) [inline net-irc/polari/polari-46.0.ebuild:32]: `telepathy-logger` was removed in `46.0`:
  - https://gitlab.gnome.org/GNOME/polari/-/blob/46.0/NEWS?ref_type=tags#L4
  - https://gitlab.gnome.org/GNOME/polari/-/merge_requests/264

**Review by Jamim — CHANGES_REQUESTED** [review]
_Hello @flint2,_

Thank you for the contribution!
However, there is room for improvement.

File: net-irc/polari/polari-46.0.ebuild (hunk 1 above)
Comment (flint2) [reply net-irc/polari/polari-46.0.ebuild]: Done in https://github.com/gentoo/gentoo/commit/7d6cff0bdf173dbed8631c88dcebe0e5cb0236b3

File: net-irc/polari/polari-46.0.ebuild (hunk 2 above)
Comment (flint2) [reply net-irc/polari/polari-46.0.ebuild]: Done in https://github.com/gentoo/gentoo/commit/7d6cff0bdf173dbed8631c88dcebe0e5cb0236b3

**Review by Jamim — APPROVED** [review]

**Pull request closed without merging**